redis.conf

```diff
@@ -1330,10 +1330,34 @@ disable-thp yes
 
 appendonly no
 
-# The name of the append only file (default: "appendonly.aof")
+# The base name of the append only file.
+# 
+# Redis can persist the data to a set of append-only files, they are divided
+# into two types, one is the the base type, it represents an initial (RDB
+# or AOF format) snapshot of the data present when the AOF is rewritten. The 
+# others are incremental type, they contain the incremental commands since the last 
+# AOF rewrite.
```

Comment:
```suggestion
# base AOF file was created.
```

Reply:
i think they mean the same thing, right?

Reply:
i think base is better.. rewrite is just one way to create a base file (see plans in #9794 )

Reply:
Can you explain in more detail "always generate base rdb (even when starting empty)"

Reply:
when rdb-preamble is enabled (or maybe even if it isn't), and redis starts up empty configured to persist to AOF, i want it to do an rdbSaveRio and generate an empty base file.

this may be required for some modules that want to persist their configuration into rdb aux fields, so that when we recover from persistence, they know with which config that persistence was created.

Reply:
OK